vigetlabs/wordpress-site-starter
# [#124] Adding btn icon function

# Summary

This adds in the function to add custom icons to the button block. 

## Issues

* #124

## Testing Instructions

1. Go into the file `wp-content/themes/wp-starter/inc/icons.php` and add a custom icon. 
2. Make sure that icons pull into the button icon set.

## Screenshots

<img width="289" alt="Screenshot 2024-07-23 at 3 56 56 PM" src="https://github.com/user-attachments/assets/7f87f437-1864-40b7-9d7f-7daa72b4a815">
